lib/internal/Magento/Framework/Locale/Resolver.php

```diff
@@ -159,4 +169,18 @@ public function revert()
         }
         return $result;
     }
+
+    /**
+     * Retrieve Deployment Config
+     *
+     * @return DeploymentConfig
+     */
+    private function getDeploymentConfig()
+    {
+        if (!$this->deploymentConfig) {
+            $this->deploymentConfig = ObjectManager::getInstance()->get(DeploymentConfig::class);
```

Comment:
Could you move deploymentConfig as last argument in constructor? It's not a good idea to get it through ObjectManager.

See "Adding a constructor parameter" section in https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/

Reply:
Also look at failing unit tests. Please fix them

Reply:
@ihor-sviziev please check again now